docs-site/docs.html

```diff
@@ -145,39 +145,242 @@ <h3>Badge on Icons</h3>
     <hr />
     <!-- Buttons -->
     <h2>Buttons</h2>
```

Comment:
here you have already given heading `h2` so no need to add comments

Reply:
Hi Bhakti, Thanks for your Review. 
At the beginning of each component code section, I write component name in the comment at start, as it acts like divider between multiple code components and helps me for easily reading the code.
Request you to please provide approval.